.travis.yml

```diff
@@ -58,4 +58,7 @@ install:
   - if [[ "$RUN_UI_TESTS" ]]; then . $HOME/.nvm/nvm.sh && cd ui && nvm use && cd .. ; fi
 
 script:
-  - sudo -E "PATH=$PATH" make travis
+  - sudo -E "PATH=$PATH" GOTESTSUM_JUNITFILE=/tmp/results.xml make travis
+
+after_script:
+  - cat /tmp/results.xml
```

Comment:
Do we need to do this right now? Is it useful enough that people will copy and paste it out of Travis?

Reply:
Yes!  At least for me ;-).  I use travis cli[1] to fetch logs and then analyze them locally to identify flaky tests.  When viewed on the TravisCI site, the output is collapsed by default, so hopefully doesn't impact others.

[1] https://github.com/travis-ci/travis.rb

Reply:
+1 from me on this being super useful. Honestly probably my most missed CCI feature is the test results handling.

Reply:
Cool! Very excited about new tools to investigate builds!